pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
+
+- Download file using remote name:
+
+`curl -O {{http://example.com/filename}}`
+
+- Automatically continue/resume previous file transfer:
+
+`curl -O -C - {{http://example.com/filename}}`
 
 - Send form-encoded data:
 
-`curl --data {{name=bob}} {{http://localhost/form}}`
+`curl --data {{name=bob}} {{http://example.com/form}}`
 
 - Send JSON data:
 
-`curl -X POST -H "Content-Type: application/json" -d {{'{"name":"bob"}'}} {{http://localhost/login}}`
+`curl -X POST -H "Content-Type: application/json" -d {{'{"name":"bob"}'}} {{http://example.com/login}}`
 
 - Specify an HTTP method:
 
-`curl -X {{DELETE}} {{http://localhost/item/123}}`
+`curl -X {{DELETE}} {{http://example.com/item/123}}`
 
 - Head request:
```

Comment:
While we are on this page, I feel we can do away with this option. Showing the `--head` option is redundant here immediately after showing the `-X METHOD` option. One can easily do a `-X HEAD` to make a head request. 

Given the limited space that we have, I think we should adhere to showing only one way to achieve something.